include/powsybl/iidm/Identifiable.hpp

```diff
@@ -22,10 +23,19 @@ namespace iidm {
 
 class Network;
 
-class Identifiable : public virtual Validable, public Extendable {
+class Identifiable : public virtual Validable, public Extendable, public MultiVariantObject {
 public: // Validable
     std::string getMessageHeader() const override;
 
+protected:  // MultiVariantObject
+    void allocateVariantArrayElement(const std::set<unsigned long>& indexes, unsigned long sourceIndex) override;
+
+    void deleteVariantArrayElement(unsigned long index) override;
+
+    void extendVariantArraySize(unsigned long initVariantArraySize, unsigned long number, unsigned long sourceIndex) override;
+
+    void reduceVariantArraySize(unsigned long number) override;
+
```

Comment:
This block is not well placed: should be in the `protected` area

Reply:
Ok done. (moved in its own protected section as it is a MultiVariantObject overriden methods)